app/controllers/favorites_controller.rb

```diff
@@ -1,19 +1,9 @@
 class FavoritesController < ApplicationController
+  before_action :require_user
+
   def index
-    recipe_ids = Favorite.where(user: current_user).pluck(:recipe_id).join(',')
+    @user = User.find(params[:user_id])
+    recipe_ids = Favorite.where(user: @user).pluck(:recipe_id).join(',')
```

Comment:
This could be refactored right? We're not really supposed to be making ActiveRecord queries in the controller

Reply:
Yeah, definitely agree with David on this one

Reply:
Agree. Extracting favorite recipe IDs should live in the Favorite model.

Reply:
(done)